config/environments/production.rb

```diff
@@ -67,6 +67,13 @@
   #   authentication: :plain
   # }
 
+  config.action_mailer.delivery_method = :mailgun
+  config.action_mailer.mailgun_settings = {
```

Comment:
this is probably fine but might be more accurate to call this file `shared_deployment_config.rb` if all envs are going to default to this